core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -534,37 +534,83 @@ async def confirm_output(
     ctx: GenericContext,
     address: str,
     amount: str,
-    title: str = "SENDING",
+    title: str | None = None,
     hold: bool = False,
     br_code: ButtonRequestType = ButtonRequestType.ConfirmOutput,
     address_label: str | None = None,
+    spending_label: str | None = None,
+    output_index: int | None = None,
 ) -> None:
-    title = title.upper()
-    if title.startswith("CONFIRM "):
-        title = title[len("CONFIRM ") :]
+    if title is not None:
+        title = title.upper()
 
-    description = f"To your {address_label}:" if address_label else "To:"
-    await confirm_value(
-        ctx,
-        title,
-        address,
-        description,
-        "confirm_output",
-        br_code,
-        verb="NEXT",
-    )
+    while True:
+        while True:
+            recipient_title = (
+                f"RECIPIENT #{output_index + 1}"
+                if output_index is not None
+                else title or "SENDING TO"
```

Comment:
`output_index` overrides the `title`. I wonder if it shouldn't be the other way around.

Reply:
Should be fixed in ba5f6490a6e802aa97a2441baef26decb8560c47